### PR TITLE
translation for manifest section

### PR DESCRIPTION
**Pull request by lisa0314**
@jondong @shaoboyan @chuan9, please help to review. Thanks!

### CODE REVIEW

**Comment by jondong** [comment]
@shaoboyan would you please help to review this translation? It would be much appreciated that we can leverage the review effort by team work, :)

**Comment by shaoboyan** [comment]
Oh, Sorry for late review. I'll give my comments.

**Comment by jondong** [comment]
hi @shaoboyan, any update on this? :)

**Comment by shaoboyan** [comment]
@jondong  Yes, and I'll finish all the comments today. :)

File: public/documentation/manifest/content_security_policy_zh.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# 内容安全策略
+
+内容安全策略领域规定了应用可以加载并执行哪些类型的内容。
```
Comment (shaoboyan) [inline public/documentation/manifest/content_security_policy_zh.md]: “领域” -> "域"

File: public/documentation/manifest/content_security_policy_zh.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+如果这个字段没有被设定，则没有内容安全策略被实施。默认情况下，这意味着一个Crosswalk应用可以从任何站点加载脚本和对象（通过`<script>`, `<object>`, `<embed>`和`<applet>`元素）。
+
+提供最小保护的建议默认值是：
+    "csp": "script-src 'self'; object-src 'self'"
+
+这个限制了这个应用只能使用来自它自身（例如，在Crosswalk中，作为应用的一部分分配）的JavaScript（通过`<script>`元素加载）和对象（通过`<object>`, `<embed>`, 和`<applet>`元素加载）。
```
Comment (shaoboyan) [inline public/documentation/manifest/content_security_policy_zh.md]: 这个限制了这个 -> 这个限制该

File: public/documentation/manifest/content_security_policy_zh.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    下文的CSP (上文推荐为最低策略):
+
+        "csp": "script-src 'self'; object-src 'self'"
+
+    禁用了“不安全”的JavaScript。这意味着任何对`eval()`的调用都不被允许。
+
+    在大多数情况下，推荐**尽量**避免使用`eval()`，因为它可能成为一个跨站点脚本攻击的载体。在一个拥有特定权限访问系统资源的应用中（Crosswalk应用采用的方式），这些可能尤其危险。
+
+    然而，你可能发现你却要尽力去使用一个依赖于`eval()`的第三方库；或者在某些情景下，使用`eval()`是最明智和实际的解决方案。在这些情况下，你可能决定适当放宽内容安全策略来允许调用`eval()`。
```
Comment (shaoboyan) [inline public/documentation/manifest/content_security_policy_zh.md]: 你可能发现你必须使用

File: public/documentation/manifest/display_zh.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+# 展示
+
+`display`字段用于为应用设定更优的*展现模式*：应用占据多大的可用展示区域，用户接口的其他元素（导航栏，其他工具栏）是否可见。 
+
+当前，下列展示模式可用：
+
+*   `"全屏"`
+
+    在这种模式下，应用的展示将充满整个屏幕，没有任何运行时的UI元素("chrome")可见。
```
Comment (shaoboyan) [inline public/documentation/manifest/display_zh.md]: UI元素("chrome")是不是调整一下为chrome UI元素？

File: public/documentation/manifest/launch_screen_zh.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Application readiness state | Preconditions
+--- | ---
+`"first-paint"` | * The first visually non-empty paint has occurred.
+`"user-interactive"` | * The first visually non-empty paint has occurred.<br>* The DOM and CSSOM have been constructed.
+`"complete"` | * The first visually non-empty paint has occurred. <br>* All the resources have been loaded.<br> **WARNING: This may take a long time, as it only triggers after all sub-resources have been downloaded.**
+`"custom"` | * The first visually non-empty paint has occurred. <br>* The ```window.screen.show()``` method was called.
+
+### `default` / `portrait` / `landscape`
+
+这些属性规定了在不同的屏幕方向上的启动画面时使用的图像和背景。如果仅仅`portrait`或者`landscape`被设定，启动画面将仅仅出现在对应的方向上（分别为横向或者纵向）。
```
Comment (shaoboyan) [inline public/documentation/manifest/launch_screen_zh.md]: double check一下，我理解。portrait = 纵向 landscape=横向。
如果是这样，括号里是不是调一下顺序比较好 （纵向或者横向）

File: public/documentation/manifest/permissions_zh.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+      "icons": [{
+        "src": "icon96.png",
+        "type": "image/png",
+        "sizes": "96x96",
+        "density": "2.0"
+        }
+      ]
+    }
+
+...并且这个`make_apk.py`命令行：
```
Comment (shaoboyan) [inline public/documentation/manifest/permissions_zh.md:33]: About  make_apk.py. @chuan9 这里可以有app-tools的更新么，因为make_apk已经废用了

File: public/documentation/manifest/permissions_zh.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+    <uses-permission android:name="android.permission.WRITE_CONTACTS"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.READ_SMS"/>
+    <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
+    <uses-permission android:name="android.permission.RECEIVE_SMS"/>
+    <uses-permission android:name="android.permission.SEND_SMS"/>
+    <uses-permission android:name="android.permission.WRITE_SMS"/>
+    <uses-permission android:name="android.permission.VIBRATE"/>
+
+注意`make_apk.py`脚本通常添加一个默认集合的permission（在上例中的第一组）。只有第二组的permission是因为在manifest中的`xwalk_permissions`字段而添加的。manifest中的permission和Android permission的对应关系在[table in the next section](#Permissions-required-by-API)中展示。
```
Comment (shaoboyan) [inline public/documentation/manifest/permissions_zh.md:61]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+# 使用manifest文件
+
+这个页面展示了在部署一个Crosswalk应用的时候如何使用`manifest.json`文件。
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: 删掉 “一个”

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+<p>一个manifest可以通过以下"modes"使用：</p>
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+1.  [配置如何为Android打包一个Crosswalk应用](#Configure-Android-packaging_zh).
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+2.  [如何将一个应用加载到一个嵌入式Crosswalk运行时环境](#Load-an-application-into-an-embedded-Crosswalk_zh).
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+<h2 id="Configure-Android-packaging_zh">配置Android打包</h2>
+
+一个Crosswalk manifest文件可以被用来作为一个应用生成Android包的基础。详细指导请参见[入门指导](/documentation/getting_started_zh.html)。
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+然而，那个指导只使用了一个基本的manifest，并没有详细地解释某些manifest字段如何影响Android包生成。下面的链接提供了关于这些的额外信息：
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md:14]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+
+* `图标`: [在打包Android上的作用](/documentation/manifest/icons_zh.html#Effect-on-Android-packaging_zh)
+
+* `permissions`: [在打包Android上的应用](/documentation/manifest/permissions_zh.html#Effect-on-Android-packaging_zh)
+如果没有被包含进`manifest.json`文件[加载进嵌入式Crosswalk中l](#Load-an-application-into-an-embedded-Crosswalk_zh)，则这些字段不会生效。
+
+<h2 id="Load-an-application-into-an-embedded-Crosswalk_zh">加载一个应用到嵌入式Crosswalk</h2>
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto, 且 “加载嵌入模式应用”

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+
+[嵌入式API](/documentation/apis/embedding_api_zh.html) 使得你可以将一个Crosswalk运行时环境嵌入到一个Android应用中。[嵌入式Crosswalk指南](/documentation/embedding_crosswalk_zh.html)解释了如何使用这个API加载一个应用的主要HTML文件到嵌入式Crosswalk。
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto,嵌入式->嵌入模式

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+
+然而，API[从manifest文件输出加载一个应用的方法](/documentation/apis/embedding_api_zh.html)也可以作为另一种选择。从manifest加载一个应用的优点是它比通过URL加载一个应用提供更多的灵活性。
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md:25]: ditto

File: public/documentation/manifest/using_the_manifest_zh.md [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+
+例如，如果你决定为你的应用修改入口点（例如，将`index.html`改名成`home.html`），你可以在manifest中进行而不用去修改任何Java应用代码。同样，如果一个新的字段对于Crosswalk manifest变得可用，你便可以利用你自己manifest中的这些字段而不用去修改任何Java代码。
```
Comment (shaoboyan) [inline public/documentation/manifest/using_the_manifest_zh.md]: ditto

File: public/documentation/manifest/xwalk_hosts_zh.md [hunk 18]
```diff
@@ -0,0 +1,48 @@
+# xwalk_hosts
+
+[XMLHttpRequest对象](http://www.w3.org/TR/XMLHttpRequest/)用于通过HTTP与远程服务器通信（一个通常称为[Ajax](http://www.adaptivepath.com/ideas/ajax-new-approach-web-applications/)的方法）。正常情况下，这些请求会被[same-origin policy](http://www.w3.org/Security/wiki/Same_Origin_Policy)限制：只有与发出请求的脚本位于相同主机的资源才可以被访问。
```
Comment (shaoboyan) [inline public/documentation/manifest/xwalk_hosts_zh.md]: ditto

File: public/documentation/manifest/xwalk_hosts_zh.md [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+
+如果你已经尝试从你的web应用访问资源，但是它们却位于不同的来源，你可能已经看到下列信息：
+
+    XMLHttpRequest cannot load http://crosswalk-project.org/. No
+    'Access-Control-Allow-Origin' header is present on the requested
+    resource. Origin 'file://' is therefore not allowed access.
+
+这表明如果提供你要访问的服务器返回正确的头部，便有可能发出这样的请求。这里提到的'Access-Control-Allow-Origin'头部是[cross-origin resource sharing specification](http://www.w3.org/TR/cors/)的一部分。跨源的资源分享使得一个客户端可以向不同源的服务器发出请求，通过需要的header提供客户端和服务器的通信。然而实际上，很多你可能想要访问的web服务（除非你自己控制它们）在没有认证的情况下均不允许跨源的资源共享。
```
Comment (shaoboyan) [inline public/documentation/manifest/xwalk_hosts_zh.md]: ditto

File: public/documentation/manifest_zh.md [hunk 20]
```diff
@@ -0,0 +1,79 @@
...
+                        "icons": [
+                              {
+                                        "src": "icon.png",
+                                                "sizes": "72x72"
+                                                      }
+                                                        ]
+}
+````
+
+下列是当创建一个模板时，由`crosswalk-build应用`生成的默认manifest。
```
Comment (shaoboyan) [inline public/documentation/manifest_zh.md]: ditto

**Comment by shaoboyan** [comment]
@jondong @lisa0314  thx and others LGTM

**Comment by lisa0314** [comment]
@shaoboyan , thanks for your review.

**Comment by jondong** [comment]
@lisa0314 any update on this PR? it seems there're still 2 comments haven't been addressed.

**Comment by jondong** [comment]
well, anyway let's merge it firstly, we can modify the make_apk related contents later. LGTM.

**Pull request merged**